rest/model/task.go

```diff
@@ -515,6 +516,7 @@ func (at *APITask) getArtifacts() error {
 		for _, file := range strippedFiles {
 			apiFile := APIFile{}
 			apiFile.BuildFromService(file)
+			apiFile.GetLogURL(env, utility.FromStringPtr(at.Id), at.Execution)
```

Comment:
It's the same pattern used for generating URLs for TestResults https://github.com/evergreen-ci/evergreen/blob/aa3b12975f68997b09cee7842fa14bea92557e99/model/testresult/testresult.go#L91

Reply:
GetLogURL for testResult is on the regular TestResult model, not a rest model.  It also doesn't lend itself to being called in BuildFromService because it's not part of an api struct that's being built anyway. 

Is there a reason why we don't want the parsley links included in other uses of BuildFromService?

Reply:
This is non-blocking, approving

Reply:
The main motivation was to prevent passing in all of these additional values every time BuildFromService is called and to keep the function signature the same.